### ISSUE TITLE
Critical: Fatal error when using Vipps express and cancelling payment

### ISSUE THREAD

**Issue by kimmenbert** [vippsas/vipps-recurring-woocommerce#93]
When using Vipps Express for singular (non subscription) purchase. Cancelling the payment in the Vipps pop up leads to an fatal error:

```
PHP Fatal error: Uncaught Error: Call to a member function get_meta() on null in /xxxxx/plugins/vipps-recurring-payments-gateway-for-woocommerce/includes/wc-vipps-recurring-helper.php:188
```

Cancelling in-app also generates this fatal error, however not visible front-end.

Why is Woo recurring even triggered at all in this case?

**Comment by Marcuzz** [comment]
Hi,

Can you share more of the error stacktrace? I need to know where the original call comes from in order to find out what causes this issue :)

You are right about the fact that this should not happen.

**Comment by Marcuzz** [comment]
This should be fixed in 869646b. You can download the update from GitHub. I will release the update on WordPress.org when we've dealt with the other issues as well 😄
